opal/clearwater/component.rb

```diff
@@ -136,15 +259,20 @@ def self.sanitize_attributes attributes
         attributes[:class_name] = attributes.delete(:class)
       end
 
-      attributes.each do |key, handler|
-        if key[0, 2] == 'on'
-          attributes[key] = proc do |event|
-            handler.call(Bowser::Event.new(event))
+      memo = {attributes: attributes[:attributes] || {} }
+      sanitized = attributes.each_with_object(memo) do |(key, value), hash|
```

Comment:
Good point. I haven't even checked `SVGComponent`'s performance the way I do with DOM components. I actually only added SVG support for a company that's building an Electron app with Clearwater because I thought it was a cool idea. I've played with it, but I suppose I should check its actual performance. :-)